src/actions.js

```diff
@@ -188,8 +188,14 @@ export const types = {
   FIRSTADDRESS_FAILURE: 'FIRSTADDRESS_FAILURE',
   /* It updates the redux state of new nano contract transaction status on wallet connect register. */
   WALLETCONNECT_NEW_NANOCONTRACT_STATUS: 'WALLETCONNECT_NEW_NANOCONTRACT_STATUS',
+  /* It triggers a process to fetch token details for a list of unregistered tokens. */
   UNREGISTEREDTOKENS_REQUEST: 'UNREGISTEREDTOKENS_REQUEST',
-  UNREGISTEREDTOKENS_UPDATE: 'UNREGISTEREDTOKENS_UPDATE',
+  /* It signals the process has loaded at least one token details with success. */
+  UNREGISTEREDTOKENS_SUCCESS: 'UNREGISTEREDTOKENS_SUCCESS',
```

Comment:
What do you think about calling this `UNREGISTEREDTOKENS_REQUEST_SUCCESS` (and the same for other actions)

I think this would be better to debug using react dev tools

Reply:
I will call it `UNREGISTEREDTOKENS_DOWNLOAD_SUCCESS` and add `DOWNLOAD` to the other actions for consistency.

Reply:
Done at: f58331b54963d2e26ea2976474046cc273875344